pages/common/bzip2.md

```diff
@@ -14,3 +14,19 @@
 - Decompress a file to standard output:
 
 `bzip2 -dc {{path/to/compressed_file.bz2}}`
+
+- Test compressed file integrity:
+
+`bzip2 -t {{path/to/compressed_file.bz2}}`
+
+- Show the compression ratio for each file processed in [v]erbose mode:
```

Comment:
```suggestion
- Show the compression ratio for each file processed with detailed information:
```

Reply:
IMO, Both the versions are fine. What should we do?